tests/test_big_int_id.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+
+from gdc_ng_models.models import released_data
+
+
+@pytest.mark.usefixtures("create_released_data_db")
+def test_big_int_id(db_session):
+    """Test the BIGINT 'id' column in specific tables
+
+    For the sake of simplicity, we only tested the released_data_log table here
```

Comment:
So have to thoughts on this:
1. Feels like this test should really be in the same file where we test other release data logs. It feels like that is more where this belongs. Generally, tests are tied to namespace (though I know we stray from that often).
2. Is there no way we could modify existing tests to insure more converge of this change with more of the objects being modified?